test/AssociationRegistry.Test.Admin.Api/Commands/VerenigingOfAnyKind/When_Wijzig_Lidmaatschap/RequestValidating/A_Invalid_Request.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace AssociationRegistry.Test.Admin.Api.Commands.VerenigingOfAnyKind.When_Wijzig_Lidmaatschap.RequestValidating;
```

Comment:
Is er een mogelijkheid om een specifieke validator te extracten? Dan hebben we mogelijks minder duplicate testen.

Reply:
Op zich moeilijke abstractie dan zouden we de requests ook moeten aanpassen vermoed ik, ik bekijk het na verwijder lidmaatschap verder